### PR TITLE
[sylius/resource-bundle] Sylius configuration for routes with attributes' feature

### PR DESCRIPTION
**Pull request by loic425**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | 

Only the routes/sylius_resource.yaml file has been added. the other files are just "copied".

### CODE REVIEW

File: sylius/resource-bundle/1.9/manifest.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "container": {
+        "locale": "en"
```
Comment (Nyholm) [inline sylius/resource-bundle/1.9/manifest.json:3]: Why is this needed?

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you. Just one comment.

File: sylius/resource-bundle/1.9/manifest.json (hunk 1 above)
Comment (vvasiloi) [reply sylius/resource-bundle/1.9/manifest.json:3]: Legacy. The bundle still uses this parameter as the default locale for translations instead of using configuration. 🙈

Comment (loic425) [reply sylius/resource-bundle/1.9/manifest.json:3]: @vvasiloi Maybe we can fix it before RC release. But I think we have to mention it on UPGRADE file, cause it can break the configuration.

Comment (vvasiloi) [reply sylius/resource-bundle/1.9/manifest.json:3]: Yes, we could deprecate it. Not sure how to configure it though. Should we define new config, or use the config of Symfony Translations? Right now it assumes that parameter is used to configure Symfony Translations.

Comment (vvasiloi) [reply sylius/resource-bundle/1.9/manifest.json:3]: @loic425 we're waiting for https://github.com/Sylius/SyliusResourceBundle/pull/374, right?

Comment (loic425) [reply sylius/resource-bundle/1.9/manifest.json:3]: Yes!

**Comment by fabpot** [comment]
Closing as there is no more activity and the PR cannot be merged as is. Feel free to reopen it if you like.
